src/safeposix/syscalls/sys_calls.rs

```diff
@@ -201,12 +270,19 @@ impl Cage {
                 }
```

Comment:
why the reference counter of the socket got incremented twice here

Reply:
It was a bug and is removed in this commit!